ibis/backends/tests/test_string.py

```diff
@@ -1012,9 +1012,21 @@ def test_parse_url(con, result_func, expected):
     assert result == expected
 
 
-def test_capitalize(con):
-    s = ibis.literal("aBc")
-    expected = "Abc"
+@pytest.mark.parametrize(
+    ("inp, expected"),
+    [
+        (None, None),
+        ("", ""),
+        ("Abc", "Abc"),
+        ("aBc", "Abc"),
+        ("aBc def", "Abc Def"),
+        ("aBc-def", "Abc-Def"),
```

Comment:
I think the expected behavior should be

```suggestion
        ("aBc def", "Abc def"),
        ("aBc-def", "Abc-def"),
```